custom_components/ramses_cc/translations/en.json

```diff
@@ -0,0 +1,183 @@
+        },
+        "error": {
+            "invalid_gateway_config": "Invalid ramses_rf config: {error_detail}",
+            "invalid_port_config": "Invalid serial port config: {error_detail}",
+            "invalid_regex": "Invalid regular expression: {error_detail}",
+            "invalid_schema": "Invalid schema: {error_detail}",
+            "invalid_traits": "Invalid device traits: {error_detail}"
+        },
+        "step": {
+            "init": {
+                "menu_options": {
+                    "choose_serial_port": "Serial port",
+                    "config": "Gateway configuration",
+                    "schema": "System schema and known devices",
+                    "advanced_features": "Advanced features",
+                    "packet_log": "Packet log",
+                    "clear_cache": "Clear cache"
+                }
+            },
+            "choose_serial_port": {
+                "title": "Choose serial port",
+                "data": {
+                    "port_name": "Serial device path"
+                }
+            },
+            "configure_serial_port": {
+                "title": "Serial port config",
+                "data": {
+                    "port_name": "Serial device path",
+                    "serial_port": "Advanced serial port config"
+                },
+                "data_description": {
+                    "serial_port": "Not required for typical use."
+                }
+            },
+            "config": {
+                "title": "Gateway configuration",
+                "data": {
+                    "scan_interval": "Scan interval",
+                    "ramses_rf": "Advanced ramses_rf gateway configuration"
+                },
+                "data_description": {
+                    "scan_interval": "How often to poll for changes in state. Values below 60 or above 180 are not recommended.",
+                    "ramses_rf": "Not required for typical use."
+                }
+            },
+            "schema": {
+                "title": "System schema and known devices",
+                "description": "Refer to the configuration section in the [wiki](https://github.com/zxdavb/ramses_cc/wiki/) for further details and examples of system schema configuration.",
+                "data": {
+                    "schema": "System schema(s)",
+                    "known_list": "Known device IDs",
+                    "enforce_known_list": "Accept packets from known devices IDs only"
+                },
+                "data_description": {
+                    "schema": "A mapping of system device IDs to their respective schemas. This should be kept minimal and only contain devices that are not automatically discovered. For example:\n```\n01:111111: # Temperature control system (e.g. evohome)\n  system:\n    appliance_control: 10:123446\n  zones:\n    \"07\": '{sensor: 01:111111}'\n\norphans_hvac: [30:111111, 32:333333, 32:555555, 32:666666]\n```",
```

Comment:
I'm unsure about adding in the examples here. They are convenient but do make the dialog even taller and require more scrolling. Perhaps the link to the wiki in the main dialog description is sufficient.

Reply:
Point to the wiki. Keep the translations thin.

Then the two are decoupled, and maintenance will be 100% easier.